scheds/rust/scx_p2dq/src/bpf/main.bpf.c

```diff
@@ -1594,6 +1600,38 @@ s32 BPF_STRUCT_OPS_SLEEPABLE(p2dq_init)
 	return p2dq_init_impl();
 }
 
+void BPF_STRUCT_OPS(p2dq_cpu_acquire, s32 cpu,
+		    struct scx_cpu_acquire_args *args)
+{
+	struct cpu_ctx *cpuc;
+	struct llc_ctx *llcx;
+	u64 now = scx_bpf_now();
+
+	if (!(cpuc = lookup_cpu_ctx(cpu)) ||
+	    !(llcx = lookup_llc_ctx(cpuc->llc_id)))
+	    return;
+
+	if (!cpuc->idle)
+		llcx->load += now - cpuc->last_acquired;
+}
+
+void BPF_STRUCT_OPS(p2dq_cpu_release, s32 cpu,
+		    struct scx_cpu_release_args *args)
+{
+	struct cpu_ctx *cpuc;
+	u64 now = scx_bpf_now();
+
+	if (!(cpuc = lookup_cpu_ctx(cpu)))
+	    return;
+
+	if (is_idle_task(args->task)) {
```

Comment:
Hm... I don't think `ops.cpu_release()` is called if we're switching to a scheduling class below SCHED_EXT. See: https://web.git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/kernel/sched/ext.c#n3171

So this condition should be always false, unless I'm missing something...